pkg/autoconf/autoconfhomeconfigfiles_test.go

```
@@ -0,0 +1,514 @@
+}
+
+func detectedGitconfig() DetectedHomeConfigFile {
+	return DetectedHomeConfigFile{
+		Name:          "gitconfig",
+		HostPath:      "$HOME/.gitconfig",
+		ContainerPath: "$HOME/.gitconfig",
+	}
+}
+
+// alwaysGlobalHomeConfig is a selectTarget stub that always picks the global target.
+func alwaysGlobalHomeConfig(_ []HomeConfigFilesConfigTargetOption) (HomeConfigFilesConfigTarget, error) {
```

golangci-lint (2.12.1)

[error] func alwaysGlobalHomeConfig is unused

(unused)